lib/iris/analysis/_scipy_interpolate.py

```diff
@@ -226,6 +226,7 @@ def compute_interp_weights(self, xi, method=None):
                 n_indices = np.ravel_multi_index(corner_indices, shape,
                                                  mode='wrap')
                 col_indices[i::n_src_values_per_result_value] = n_indices
+                del(n_indices)
```

Comment:
Would this be simpler if you just assigned the `np.ravel` call to the slice of `col_indices` directly without the intermediate `n_indices`?